lib/helpers/time.js

```diff
@@ -53,6 +85,19 @@ function getDisplayFormat(time) {
   return moment.tz(time, timezone).format('h:mm a z');
 }
 
+function getDisplayFormatForDays(days) {
+  if(days.length === 5) {
+    return 'all weekdays';
+  }
+
+  return days.reduce((previous, current, index) => {
+    if(index == days.length - 1 && days.length > 1) {
+      return `${previous} and ${current}`;
+    }
+    return `${previous}${previous.length > 0 ? ', ' : ''}${current}`;
+  }, '');
```

Comment:
Good call.  Javascript doesn't support negative indices or range operations, but this is nearly as good:

```javascript
if (days.length <= 1) {
  return days[0] || 'no days';
}

let last = days[days.length - 1];
let rest = days.slice(0, days.length - 1);

return rest.join(', ') + ' and ' + last;
```